.env.sample

```diff
@@ -0,0 +1,44 @@
+ENVIRONMENT=dev
+SECURITY_KEY=
+
+# For Craft Postgres DB
+DB_DSN=pgsql:host=postgres;dbname=craft
+DB_SERVER=postgres
+DB_NAME=
+DB_USER=
+DB_PASSWORD=
+DB_DATABASE=
+DB_SCHEMA=public
+DB_DRIVER=pgsql
+DB_PORT=5432
+
+ENABLE_MEMCACHED="true"
+MEMCACHED_IP=cache
```

Comment:
This works fine without specifying the MEMCACHED_PORT? There is a default port defined in the linked file, but just thought I'd double-check

https://github.com/lsst-epo/rubin-obs-api/blob/develop/api/config/app.php#L36